docs/guides/installation.md

```diff
@@ -217,7 +217,15 @@ flux bootstrap gitlab \
 
 ### Generic Git Server
 
-For other Git providers such as Bitbucket, Gogs, Gitea, etc you can manually setup the repository and the deploy key.
+For other Git providers such as Bitbucket, Gogs, Gitea, Azure DevOps, etc you can manually setup the repository and the deploy key.
+
+#### Note about Azure DevOps
+Azure DevOps requires a specific git implementation to be used that the other git providers do not. This is due to the Azure DevOps
+git server expects capabilities that have not been implemented in the default got implementation used by Flux. Enabling this option
+will introduce certain side effects that you can read about in the Source Controller documentation.
+Additionally the current implementation of image automation does not support Azure DevOps as it also has a git client that does not
+support Azure Devops. This can change in the future, but would require an update that would add an option to use a different git
+implementation for Azure DevOps support.
```

Comment:
```suggestion

Azure DevOps requires a non-default Git implementation (`libgit2`) to be enabled, so that the Git v2 protocol is supported. Note that this implementation does not support shallow cloning, and it is therefore advised to only resort to this option if a connection fails with the default configuration.
```

Reply:
NB: just a mere suggestion, as I had some trouble gathering all the information from what was written there.

Reply:
Yours sounds better so I made the change :)